lib/Gateway.js

```diff
@@ -621,6 +621,26 @@ function setDiscoveryValue (payload, prop, node) {
   }
 }
 
+/**
+ *
+ * @param {*} node node payload
+ * @param {*} valueId the valueId or CFG
+ * @param {*} nodeName compiled nodeName
+ * @param {*} valueId node value info
```

Comment:
Add types

Reply:
Will do, still working on function :) thnx

Reply:
thank you @robertsLando for fixing this